tests/test_cli/test_misc.py

```diff
@@ -85,6 +85,7 @@ def test_flag_help():
   push              Push item to Registry or save it in local packages.
   register          Register a new Registry account.
   remove            Remove a resource from the agent.
+  reset_password    Reset password of Registry account.
```

Comment:
@DavidMinarsch @panasevychol in the long term, I propose having a subcommand `auth` with

Reply:
I agree. We can have a ticket for that. If we have plans to add more auth related commands it would be nice to organize that way.